python_modules/dagster/dagster/_core/definitions/declarative_scheduling/serialized_objects.py

```diff
@@ -106,6 +110,41 @@ def from_result(result: "SchedulingResult") -> "AssetConditionEvaluation":
             ],
         )
 
+    def get_result_hash(self) -> str:
+        """Returns a unique string identifier which remains stable across processes to determine
+        if anything has changed since the last time this has been evaluated.
+        """
```

Comment:
should this have its own dedicated test setup that compares against persisted values from past processes?